launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -48,6 +48,36 @@ void CModListView::changeEvent(QEvent *event)
 	QWidget::changeEvent(event);
 }
 
+void CModListView::dragEnterEvent(QDragEnterEvent* event)
+{
+	event->acceptProposedAction();
+}
```

Comment:
According to Qt docs we should indicate whether we accept event:
```
 void Window::dragEnterEvent(QDragEnterEvent *event)
 {
     if (event->mimeData()->hasFormat("text/plain"))
         event->acceptProposedAction();
 }
```
( I guess in our case we should check `mimeData->hasUrls()` and look up whether there are any maps/zips/etc)